### DISCUSSION TITLE
How do I remove 2FA verification from my personal account

### DISCUSSION THREAD

**Discussion by CLXiao-1029** [community/community#131644] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I remove 2FA verification from my personal account? The code in the repository is a backup of notes, and it is too cumbersome to verify 2fa frequently for these things.
如何删除我个人账户中的2FA验证？仓库中的代码本来就是笔记备份，为了这些东西来频繁的验证2fa实在过于麻烦。

**Comment by gaomingzhao666** [comment]
Hey bro.

**There is a documentation about [how to disable 2FA auth](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account).**

**If you think the process of authentication is too upset, you can try some extension to ease that procedure, such as use [chrome extension authenticator](https://chromewebstore.google.com/detail/authenticator/bhghoamapcdpbohphigoooaddinpkbai)**

If my comment is help you solve this question, please mark my comment as answer, thanks bro. 😊

**Comment by TheGiraffe3** [comment]
You no longer can. 2FA is now mandatory on all GitHub accounts.
It started on the 7th.

**Comment by ThatOSDev** [comment]
Well I personally will just simply stop using Github then. I hate that 2FA stuff.

EDIT : There are other options that does NOT force "GIVE ME YOUR INFO" like Microcrap does !

**Comment by George-cb** [comment]
I am trying to have PLAID connect to my accounts. To link bank account to my exsisting External Bank for Credit Karma Save Accout. My funding account with Park National Bank was Closed and a new account replaced it.
